### PR TITLE
Get params from storage not from config

### PR DESCRIPTION
**Pull request by dafeda**
## Pre review checklist

- [ ] Read through the code changes carefully after finishing work
- [ ] Make sure tests pass locally (after every commit!)
- [ ] Prepare changes in small commits for more convenient review (optional)
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: tests/unit_tests/analysis/test_es_update.py [hunk 1]
```diff
@@ -36,18 +36,25 @@ def remove_timestamp_from_logfile(log_file: Path):
...
     """
     Note that this is now a snapshot test, so there is no guarantee that the
     snapshots are correct, they are just documenting the current behavior.
     """
     ert = snake_oil_case_storage
     es_update = ESUpdate(ert)
+    prior_ens = snake_oil_storage.get_ensemble_by_name("default_0")
+    experiment = snake_oil_storage.get_experiment(prior_ens.experiment_id)
+    posterior_ens = snake_oil_storage.create_ensemble(
+        experiment,
```
Comment (oyvindeide) [inline tests/unit_tests/analysis/test_es_update.py:49]: Think you can just do: prior_ens.experiment_id here

Comment (oyvindeide) [reply tests/unit_tests/analysis/test_es_update.py:49]: Same for the other instances as well

**Review by oyvindeide — APPROVED** [review]
LGTM! Just some minor comments wrt tests

File: tests/unit_tests/analysis/test_es_update.py (hunk 1 above)
Comment (dafeda) [reply tests/unit_tests/analysis/test_es_update.py]: Fixed, thanks 👍

**Pull request merged**